src/modules/specialists/components/Specialist/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Icon } from '../../../core'
+
+const Specialist = ({ person }) => (
+  <article className="specialist">
+    <div>
+      <img src={person.picture} className="specialist__picture" />
```

Comment:
Probably needs an alt attribute, is there acceptable text already?  I almost want to recommend making the whole `<article>` here a `<figure>` tag with the `<section>` being the `<figcaption>`.

Reply:
Hmm that's Interesting. I enjoy digging into html semantics. Let's see:

> Usually a `<figure>` is an image, illustration, diagram, code snippet, etc., that is referenced in the main flow of a document, but that can be moved to another part of the document or to an appendix without affecting the main flow.

> When an `<article>` element is nested, the inner element represents an article related to the outer element. For example, the comments of a blog post can be <article> elements nested in the <article> representing the blog post.

Copied lines from MDN.

I forgot to add `alt=""`, whoops. That's a mistake. 

I see how `<figure>` would work since it's related to the document, but I wonder if it loses some of it's relation to the heading `<h1>Talk to a Library Specialist</h1>` that comes with a having a parent `<article>`. I don't think the image here in this case is important. The primary content is if the person's metadata (job title) is related to their research interest. Perhaps making this a `<figure>` puts too much weight on the image.

```html
<article>
  <h1>Talk to a Library Specialist</h1>

  <article>
    <img alt="" />
    <h1>Scott Dennis</h1>
    ... metadata
  </article>
</article>
```

VS.

```html
<article>
  <h1>Talk to a Library Specialist</h1>

  <figure>
    <img alt="" />
    <figcaption>
        ... heading
        ... metadata
    </figcaption>
  </figure>
</article>
```

Reply:
I've thought about this for awhile and after staring at both, they have lost all meaning :) I could see the use of the `<figure>` and `<figcaption>` until I started reading the Mozilla documentation again. Is the nested `<article>` meant in the above example? I preferred the original with the `<section>` in the `<article>`.

Reply:
Yeah, the nested `<article>` is intentional.

The example MDN gives is a blog post `<article>` with nested comment `<articles>`. In this case it's a single library specialists `<article>` with many `<article>` specialists nested inside.

Reply:
I could see how a `<section>` would work too. Are library specialists on their own a "self-contained composition in a document," meaning it should be an `<article>`?

```html
<article>
  <h1>Talk to a Library Specialist</h1>

  <section>
    <img alt="" />
    <h2>Scott Dennis</h2>
    ... metadata
  </section>
</article>
```

Reply:
I keep thinking about options for this ...

Taking from the `<article>` MDN example: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article

```html
<article>
  <h1>Talk to a Library Specialist</h1>

  <section>
    <article>
      <img alt="" />
      <h2>Scott Dennis</h2>
      ... metadata
    </article>
    <article>
      ...
    </article>
  </section>
</article>
```